Project.toml

```diff
@@ -7,11 +7,14 @@ version = "0.1.0"
 BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+JuliaFormatter = "98e50ef6-434e-11e9-1051-2b60c6c9e899"
+LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
```

Comment:
I think these two packages are for test only? If so, we can create a new `Project.toml` file under `test` folder.